resources/migrations/permissions/rollback/view_data.sql

```diff
@@ -118,3 +118,11 @@ WHERE pg.name != 'Administrators'
      WHERE dp.group_id = pg.id
        AND dp.table_id = mt.id
        AND dp.perm_type = 'perms/create-queries');
+
+DELETE
+FROM data_permissions
+WHERE perm_type = 'perms/view-data';
+
+DELETE
+FROM data_permissions
+WHERE perm_type = 'perms/create-queries';
```

Comment:
This is unrelated, but this makes sure we clean up the new permission types in the rollback. This doesn't really matter since if you rollback all the way to 49 we delete the `data_permissions` table entirely anyway. But it makes things more consistent if you rollback only to this migration and then migrate back up again.